libs/designsystem/src/lib/components/chart/chart-helper.ts

```diff
@@ -27,6 +18,8 @@ export class ChartHelper {
   }
 
   private renderChart(options: Options) {
-    Highcharts.chart(this.chartContainer.nativeElement, options);
+    if (this.chartContainer) {
+      this.chart = Highcharts.chart(this.chartContainer.nativeElement, options);
```

Comment:
Is this used anywhere? If not, delete:
```suggestion
      Highcharts.chart(this.chartContainer.nativeElement, options);
```